decidim-core/app/validators/etiquette_validator.rb

```diff
@@ -0,0 +1,41 @@
+# coding: utf-8
+# frozen_string_literal: true
+# This validator takes care of ensuring the validated content is
+# respectful, doesn't use caps, and overall is meaningful.
+class EtiquetteValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    return if value.blank?
+
+    validate_caps(record, attribute, value)
+    validate_marks(record, attribute, value)
+    validate_long_words(record, attribute, value)
+    validate_caps_first(record, attribute, value)
+  end
+
+  private
+
+  def validate_caps(record, attribute, value)
+    return if value.scan(/[A-Z]/).length < value.length / 3
```

Comment:
I don't know, it's a convention. If we find a use case where we might want to override that value, we can discuss it. I see no point on adding an option for that.